engine/worldmap.py

```diff
@@ -17,10 +18,23 @@ def __init__(self, name = "Unnamed", description = "An empty place.", taglist =
         self.tags = taglist
         self.name = name
         self.description = description
+        self.local_items = []
+        self.item_descriptions = []
 
     def toString(self):
         return self.name + "\n" + self.description
 
+    def add_item(self, new_item):
+        self.local_items.append(new_item)
+
+    def reveal_items(self):
+        for thing in self.local_items:
```

Comment:
This works, here's a challenge if you accept:
Can you do this in one line with a list comprehension?